gitlab.go

```diff
@@ -83,6 +83,7 @@ type BuildStateValue string
 // These constants represent all valid build states.
 const (
 	Pending  BuildStateValue = "pending"
+	Created  BuildStateValue = "created"
```

Comment:
I don't think we can reuse these constants, but instead we should create dedicated `DeploymentStatus` type for this.

Reply:
Fair enough. As an aside, and I don't think this is something that can be addressed easily, but the names of these constants should contain their type (eg. `BuildStateValuePending`) to keep the API unambiguous.

Reply:
Yes, that is very true! Guess the package has grown quite a bit since it was first created and this was never addressed...

Reply:
I've namespaced my bit.